deploy/LOCAL_DEVELOPMENT_INSTALL.md

```diff
@@ -82,6 +82,7 @@ export ANALYST_USER_GROUP=TGG_Users
 export PM_USER_GROUP=TGG_PM
     
 # Set the client ID and secret for the seqr-local OAuth client (from GCP)
+# Note: do not use the values from `seqr-secrets` in secret manager, the local credentials are saved [here](https://console.cloud.google.com/apis/credentials?project=seqr-project).
```

Comment:
I initially fetched these values in the same way the infrastructure does... which was wrong.  Took a bit of poking to figure out why.